src/roktManager.ts

```diff
@@ -0,0 +1,83 @@
+export interface IRoktPartnerAttributes {
+    [key: string]: string | number | boolean | undefined | null;
+}
+
+export interface ISelectPlacementsOptions {
+    attributes: IRoktPartnerAttributes;
+    identifier?: string;
+}
+
+export interface ISelection {
+    placementId?: string;
+    status?: string;
+    error?: string;
+}
+
+export interface IRoktLauncher {
+    selectPlacements: (options: ISelectPlacementsOptions) => Promise<ISelection>;
+}
+
+export interface IRoktMessage {
+    methodName: string;
+    payload: any;
+}
+
+export default class RoktManager {
+    private launcher: IRoktLauncher | null = null;
```

Comment:
should this be public?  the reason we put this here also is so that a customer could access it right?